### DISCUSSION TITLE
About the industrial learning

### DISCUSSION THREAD

**Discussion by vishalTondale** [github/india#191] [Students]
I have watched vidios but then also it shows the status incomplete. 
What will be the problem. Can anyone know?

**Comment by extinctsion** [comment]
same with me. I watched half the video and it was auto paused. the site was not working properly so i had to refresh the page which cost me to rewatch the entire videos again. Please fix this!

**Comment by TheSuranaverse** [comment]
**FUTURE READY SEPARATE CHANNEL**

For smooth communication & discussion!
Link: https://discord.gg/ceMXzhfaka

Thank you 🙏
